### PR TITLE
Augmented the baseball example with supporting text and synthesis

### PR DESCRIPTION
**Pull request by fonnesbeck**
Fleshed out this example a little.

### CODE REVIEW

**Comment by fonnesbeck** [comment]
Odd that one of my equations is not rendering in the notebook preview. It is there in the file, looking at the raw json.

File: notebooks/baseball.ipynb [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
```
Comment (ericmjl) [inline notebooks/baseball.ipynb]: <p>Just a quick question: is it the right place in the notebook to explain why the <code>[-1::-1]</code> exists? Apart from this one little optional nitpick, I don't have other comments on the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by ColCarroll** [comment]
Thanks for finishing this!

**Comment by twiecki** [comment]
I'd prefer if someone with more experience with baseball and Bayesian statistics could review this, don't trust @fonnesbeck on this ;).

**Comment by fonnesbeck** [comment]
Don't feed the trolls.

**Pull request merged**